TASK-3/TASK-3/User.cs

```diff
@@ -0,0 +1,9 @@
+namespace TASK_3;
+
+public abstract class User
+{
+    public static string Name;
+    public static string Lastname;
+    public static DateTimeOffset BirthDateTime;
+    public static bool IsEligibleToDrive = true;
+ }
```

Comment:
Dont mark as abstract, if you are not using functionality of abstract classes. 
Fields should be private